src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java

```diff
@@ -111,46 +114,28 @@ public CaseDetails respondToApplication(String authorization, RespondToApplicati
         GenericTseApplicationTypeItem appToModify = TseApplicationHelper.getSelectedApplication(
             caseData.getGenericTseApplicationCollection(), request.getApplicationId()
         );
+
         if (appToModify == null) {
             throw new IllegalArgumentException("Application id provided is incorrect");
         }
 
+        String copyToOtherParty = request.getResponse().getCopyToOtherParty();
         GenericTseApplicationType appType = appToModify.getValue();
 
-        respondToRequestForInfo(appType);
-
-        TseApplicationHelper.setRespondentApplicationWithResponse(
-            request,
-            appType,
-            caseData,
-            caseDocumentService
-        );
-
-        createPdfOfResponse(authorization, request, caseData, appType);
+        boolean isRespondingToTribunal = request.isRespondingToRequestOrOrder();
+        if (isRespondingToTribunal) {
+            appType.setApplicationState(IN_PROGRESS);
+            appType.setClaimantResponseRequired(NO);
+        }
 
-        CaseDataContent content = caseDetailsConverter.caseDataContent(startEventResponse, caseData);
-        CaseDetails caseDetails = caseService.submitUpdate(
-            authorization,
-            request.getCaseId(),
-            content,
-            request.getCaseTypeId()
-        );
+        sendResponseToApplicationEmails(appType, caseData, caseId, copyToOtherParty, isRespondingToTribunal);
 
-        sendResponseToApplicationEmails(appType, caseData, request.getCaseId(),
-                                        request, request.isRespondingToRequestOrOrder());
+        TseApplicationHelper.setRespondentApplicationWithResponse(request, appType, caseData, caseDocumentService);
 
-        return caseDetails;
-    }
+        createAndAddPdfOfResponse(authorization, request, caseData, appType);
 
-    /**
-     * If claimant is replying to a request for info from the tribunal, update app state to inProgress and mark
-     * response as no longer required.
-     */
-    static void respondToRequestForInfo(GenericTseApplicationType appType) {
-        if (YES.equals(appType.getClaimantResponseRequired())) {
-            appType.setApplicationState(IN_PROGRESS);
-            appType.setClaimantResponseRequired(NO);
-        }
+        return caseService.submitUpdate(
+            authorization, caseId, caseDetailsConverter.caseDataContent(startEventResponse, caseData), caseTypeId);
```

Comment:
just a warning - you're likely to get merge confs on this soon - we're reworking this method to include a lot more logging this sprint.